cnf/ext/junit.bnd

```diff
@@ -43,6 +43,7 @@ junit-osgi: \
 	org.osgi.service.resolver;version=latest,\
 	org.osgi.util.function;version=latest,\
 	org.osgi.util.promise;version=latest,\
+	org.osgi.test.assertj.framework;version=latest,\
```

Comment:
Shouldn't `org.osgi.test.junit5` also be in this list?

Reply:
Yes, this was a bit messy. `bndtools.core.test` was not even using the `junit-osgi` macro. I have tidied this up now and added `assertj.promise` too for completeness.

Originally I added `assertj.framework` because I wanted to get access to the Dates functionality for the assertj-eclipse.resources library I started building (currently embedded in `bndtools.core.test`). However, I realised that `1.1` has not been released yet so I couldn't get it... at the moment, the bit I need is cut-and-pasted into `bndtools.core.test`. So strictly, I still don't need it yet. However, I'd like to refactor this once osgi-test 1.1 is released.